models/topic.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const schema = new Schema({
+    afterClass: Number,
+    monday: Number,
+    wednesday: Number,
+    thursday: Number
```

Comment:
I think we still need the following fields: beforeClass, afterClass, friday, and title.